Public/Src/Engine/Cache/Log.cs

```diff
@@ -129,5 +129,14 @@ internal abstract partial class Logger
             EventTask = (ushort)Events.Tasks.Storage,
             Message = "Retry on loading and deserializing metadata: Succeeded: {succeeded} | Retry count: {retryCount}")]
         internal abstract void RetryOnLoadingAndDeserializingMetadata(LoggingContext loggingContext, bool succeeded, int retryCount);
+
+        [GeneratedEvent(
+            (int)EventId.RetryTryOpenAndTrackPathAsync,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Warning,

```

Comment:
>Level.Warning, [](start = 25, length = 14)

Should this be a warning? WDG doesn't like warning because they treat warning as error. I suggest we change it to verbose.

Reply:
Removed.